kayttooikeus-domain/src/main/java/fi/vm/sade/kayttooikeus/model/Henkilo.java

```diff
@@ -21,12 +21,7 @@
         }
     )
 })
-public class Henkilo implements Identifiable {
-
-    @Id
-    @Column(name = "id", unique = true, nullable = false)
-    @GeneratedValue
-    private Long id;
+public class Henkilo extends IdentifiableAndVersionedEntity {
```

Comment:
Uudelleenrekisteröinti ei toimi tämän muutoksen jälkeen niin kauan kun kanta on yhteinen oppijanumerorekisterin kanssa (kts. https://github.com/Opetushallitus/kayttooikeus/commit/100cd427aef044b015f4fb68210987e0f1d267ea).

Versionumerosta ei ole tällä hetkellä mitään hyötyä koska sitä ei paljasteta ulospäin (optimistisen lukituksen kannalta) ja pelkästään kannassa sen säilömisellä en näe kovinkaan suurta hyötyä (kertoo lähinnä monta kertaa riviä on saatettu päivitellä).

Ehdottaisin että käyttöoikeuspalvelun puolella otetaan toi versionumerokenttä pois (kuten jo olikin), jotta ainakin uudelleenrekisteröinti toimii sekä yhteisellä että jaetulla kannalla.

Reply:
Laitetaan takaisin.

Reply:
init-skriptistä pitää ottaa vielä sarake pois

Reply:
Otin sieltäkin pois. Laitoin henkilö-modeliin kommenttiin muistutuksen, että muistaa lisätä myös skriptiin kun tuo joskus lisätään.

Reply:
Miksi se lisättäisiin joskus?

Reply:
Itse ainakin pidän sitä hyvänä toimintona mistä näkee heti, onko joku käpistellyt katsottavaa riviä mutta selviäisin varmaan ilmankin.

Reply:
Ok, kommentti on kuitenkin hieman pielessä koska nyt kannan splittauksessa tulee version-sarake pudottaa käyttöoikeuspalvelun kannasta (jotta uusien rivien tallennus toimii) joten jos sarake joskus lisätään takaisin pitää tehdä normaalisti uusi migraatio (eikä säätää historiaa).